packages/hash/frontend/src/pages/type-editor/index.page.tsx

```diff
@@ -104,14 +97,6 @@ const ExampleUsage = ({ accountId }: { accountId: string }) => {
         </Button>
 
         <div>
-          <TextField
```

Comment:
I didn't bother to replace the way to update the URI as the type-editor is being worked upon now. The create button still works for first click so I think it's okay?